HealthcareSystem/Backend/Model/NotificationSurveyAndFeedback/Survey.cs

```diff
@@ -34,7 +32,28 @@ public class Survey
         public string DoctorJmbg { get; set; }
         public virtual Doctor Doctor { get; set; }
 
+
+        public Survey(int behaviorOfDoctor, int doctorProfessionalism, int gettingAdviceByDoctor, int availabilityOfDoctor, 
```

Comment:
When you have time, look at the suggestion on [this pr](https://github.com/PSW-2020-ORG4/HealthcareSystem/pull/61#issuecomment-731608130) and consider some refactoring.